neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -25,6 +25,15 @@ internal GasToken()
         {
         }
 
+        internal override bool Initialize(ApplicationEngine engine)
+        {
+            if (!base.Initialize(engine)) return false;
+            if (TotalSupply(engine.Snapshot) != BigInteger.Zero) return false;
+            UInt160 account = Contract.CreateMultiSigRedeemScript(Blockchain.StandbyValidators.Length / 2 + 1, Blockchain.StandbyValidators).ToScriptHash();
+            Mint(engine, account, 30_000_000 * Factor);
```

Comment:
Why 30M here, and where will it go?

Reply:
I think that it is a random value that will be updated when launching the genesis block of NEO 3, is it right, Erik?

Reply:
It's not a random value. We must unlock 30M gas for token swap when NEO3 MainNet launched.

Reply:
But where is this value calculated? Why 30 million?